extensions/SyntaxHighlight_GeSHi/SyntaxHighlight_GeSHi.class.php

```diff
@@ -280,7 +280,9 @@ public static function prepare( $text, $lang ) {
 		$geshi->enable_keyword_links( false );
 
 		// Wikia change start
-		if( $wgTitle instanceof Title && EditPageLayoutHelper::isCodePage( $wgTitle ) ) {
+		if( $wgTitle instanceof Title && EditPageLayoutHelper::isCodePage( $wgTitle )
```

Comment:
You don't need ```EditPageLayoutHelper::isCodePage( $wgTitle )``` any more since it's a part of ```EditPageLayoutHelper::isCodeSyntaxHighlightingEnabled( $wgTitle )```